src/axom/sidre/core/DataStore.cpp

```diff
@@ -129,6 +129,40 @@ DataStore::~DataStore()
   }
 }
 
+/*
+ *************************************************************************
+ *
+ * Rewire Conduit Message Handlers to SLIC.
+ *  ... Not the best function name, but it is descriptive :-) 
+ *
+ *************************************************************************
+ */
+void DataStore::wireConduitMessageHandlersToSLIC()
+{
+  // Provide SLIC message handler functions to Conduit to log
+  // internal Conduit info, warning, error messages.
+  conduit::utils::set_error_handler( DataStoreConduitErrorHandler );
+  conduit::utils::set_warning_handler( DataStoreConduitWarningHandler );
+  conduit::utils::set_info_handler( DataStoreConduitInfoHandler );
+}
+
+/*
+ *************************************************************************
+ *
+ * Restore Conduit Message Handlers to Conduit Defaults.
+ *  ... Not the best function name, but it is descriptive :-) 
+ *
+ *************************************************************************
+ */
+void DataStore::restoreConduitDefaultMessageHandlers()
```

Comment:
"restore" bothers me a little.  I'd rather "set" or "wire".  (Just a naming issue, probably not important...)

Reply:
I agree `set` or `wire` is probably better, let's see if other folks have strong opinions.

Reply:
Since this function name is not into the whole brevity thing, how about `resetConduitMessageHandlersToDefault()` ?
(or maybe just `resetConduitMessageHandlers()` )

Reply:
Is it possible a user might have set the conduit handlers to something non-default and you are inadvertently removing their custom handler?  Like so:

```
// user set to some custom handler
...
// wireConduitMessageHandlersToSLIC()
// Ascent does its thing
// restoreConduitDefaultMessageHandlers()
...
// user no longer has a custom handler
```

Reply:
@white238  that is possible -- we could implement a push-pop style, but honestly folks can also do that themselves at the same (coarse) level.

I think `setConduitMessageHandlersToDefault()` is the most clear reflection of the logic. I will go with that and `setConduitMessageHandlersToSLIC()`